Python Tutorial Exercises(Week 3)/#2.py

```diff
@@ -0,0 +1,76 @@
+'''
+This script prints a list of all prime numbers found between a given range of x and y inclusively. 
+This script includes a while loop and a for loop.
+The outer loop tells us all the numbers in the given range and 
+the inner loop helps us find all prime numbers.
+Create a new list 'z' to store the values we find from the inner loop.
+'''
+x = 20
+y = 47
+prime_number_list = []
+u = x
+
+# for loop
+for i in range(x,y+1):
+    for a in range(2,i):
+        if i%a == 0:
+            break
+    else:
+        prime_number_list.append(i)
+print(prime_number_list)
+
+#while loop
+# set range
+while u >= x and u <= y:
+    count = 0
+    #q = 2 is used for checking the factor of the number
+    # we start dividing by 2. 
+    # This is because divide 0 is invalid and any number can be divided by 1.
+    # So we will make a variable with the starting value of 2.
+    q = 2
+    
+    #use a while loop to calculate the prime number. 
+    # The smallest prime number is 2. the largest factor for q must be 
+    # smaller and equal to u/2, and check all integers smaller or equal 
+    # or smaller and equal to u/2. 
+    while q <= u/2:
+        #We are dividing the number by all the numbers using u%q == 0
+        if u%q == 0:
+            count += 1      
```

Comment:
As I've previously commented, do you need this `count` variable? Have a look at your own solution from line 17 in this same file